dojo/models.py

```diff
@@ -409,6 +409,11 @@ class System_Settings(models.Model):
         blank=False,
         verbose_name='Enable user profile for writing',
         help_text="When turned on users can edit their profiles")
+    enable_product_tracking_files = models.BooleanField(
+        default=True,
+        blank=False,
```

Comment:
I believe when you set blank=False for a boolean field it will not allow the checkbox to be left blank.

Reply:
I may have this confused with required looking at the file in it's totality, going to go ahead and approve.